tests/unit/CrowdingComponent.spec.js

```diff
@@ -0,0 +1,36 @@
+import { shallowMount } from "@vue/test-utils";
+import CrowdingComponent from "@/components/CrowdingComponent.vue";
+
+describe("CrowdingComponent", () => {
+  describe("Methods", () => {
+    test("searchLocation should return the search location and set haveSearched to true", () => {
```

Comment:
I see it checking for the location, but not checking the `haveSearched`

Reply:
True, I've since added to the test!